### DISCUSSION TITLE
How to prepare an apk file from the aar file?

### DISCUSSION THREAD

**Discussion by AzuxirenLeadGuy** [hajimehoshi/ebiten#2938] [Q&A] [answered]
Hi. Congratulations to all the developers who have helped in making this wonderful project.

I am following [official guide](https://ebitengine.org/en/documents/mobile.html) for making an Android app. I have successfully prepared an `aar` file with a `jar` file. I am not sure how to proceed further in preparing an APK out of it. Sorry if it is a silly question, but this process should also be included in the documentation.

**Comment by hajimehoshi** [comment]
You have to create your own Android project. As an example, see https://github.com/hajimehoshi/go-inovation

**Reply by AzuxirenLeadGuy** [reply]
Thanks for the quick response.

Creating such a project via Android studio the only way for achieving this, or is it possible to quickly generate such a project in command-line?

**Reply by hajimehoshi** [reply, marked as answer]
Creating a project via Android Studio is the only way so far. I don't plan to implement a command to generate a while project.
